src/pages/registration/submission_tab/submission_contributors.tsx

```diff
@@ -21,14 +21,15 @@ const SubmissionContributors: React.FC = () => {
   return (
     <LabelContentRow label={t('heading.contributors')} multiple>
       {contributors.map((contributor) => (
-        <NormalText key={contributor.identity.name}>
+        <Typography key={contributor.identity.name}>
+          {contributor.correspondingAuthor && <span>{t('submission.corresponding_author')}: </span>}
```

Comment:
det skal være så basic i første omgang. tror kine ser på en bedre løsning på sikt (for hele denne taben i grunn)